README.md

```diff
@@ -247,6 +247,8 @@ slaves:
     remote_proto:		http(default) or https
     default_slave:		Does this slave is the default slave where to add boards (default: lab-slave-0)
     export_ser2net:		Do ser2net ports need to be available on host
+    export_port:		Export ports p1 on the host to p2 on the worker slave.
```

Comment:
Hmm, as I think about it, "export" isn't the right word either, your first patch was better.  I think "expose" is better because that's the term docker docs use.   So..

```
expose_ports:  Expose container port p2 to port p1 on the host`
 - p1:p2
```